services/cases-api/lambdas/updateCase.js

```diff
@@ -88,25 +102,22 @@ export async function main(event) {
   }
 
   const caseKeys = {
-    PK: `USER#${personalNumber}`,
-    SK: `CASE#${id}`,
+    PK: userCase.PK,
+    SK: userCase.SK,
   };
 
-  const params = {
+  const updateCaseParams = {
     TableName: config.cases.tableName,
-    Key: {
-      PK: caseKeys.PK,
-      SK: caseKeys.SK,
-    },
-    UpdateExpression: UpdateExpression.join(', '),
+    Key: caseKeys,
```

Comment:
Do we need to break out the caseKeys? Feels like it would be easier to read if the key "Key" in updateCaseParams just defines the case keys

```JS
const updateCaseParams = {
  ...other params,
  Key: {
    PK: userCase.PK,
    SK: userCase.SK,
  }
}
```

Reply:
...other params?! what is this?